packages/gatsby-remark-copy-linked-files/README.md

```diff
@@ -163,53 +166,44 @@ destinationDir: f => `${f.hash}`
 **Error thrown:**
 
 ```js
-# cannot save outside root directory (i.e., outside `public` folder)
+# cannot save outside `public` folder
 destinationDir: `../path/to/dir`
 destinationDir: _ => `../path/to/dir`
 destinationDir: f => `../path/to/dir/${f.name}`
 destinationDir: f => `../${f.hash}`
 ```
 
----
-
 ### Custom set which file types to ignore using `ignoreFileExtensions`
 
-By default, the file types that this plugin ignores are: `png`, `jpg`, `jpeg`, `bmp`, `tiff`.
-
-> For example, `[Download it now](image.png)` will be ignored and not copied to the root dir (i.e. `public` folder)
+By default, the file types that this plugin ignores are: `png`, `jpg`, `jpeg`, `bmp`, `tiff`. For example, `[Download it now](image.png)` will be ignored and not copied to the root of the `public` folder.
```

Comment:
Same thing as the other suggestion - since "root" / "root directory" was removed elsewhere, maybe we should also remove it here:

```suggestion
By default, the file types that this plugin ignores are: `png`, `jpg`, `jpeg`, `bmp`, `tiff`. For example, `[Download it now](image.png)` will be ignored and not copied to the `public` folder.
```